### PR TITLE
PKG-5552: Update sympy dependency to 1.13.2

### PR DESCRIPTION
**Pull request by jnoller**
sympy 1.13.2 

**Destination channel:** defaults

### Links

- [PKG-5552](https://anaconda.atlassian.net/browse/PKG-5552) 
- [Upstream repository](https://github.com/conda-forge/sympy-feedstock)
- [Upstream changelog/diff](https://github.com/conda-forge/sympy-feedstock/commits/main/)

### Explanation of changes:

- Update version / build versions to match upstream feedstock

[PKG-5552]: https://anaconda.atlassian.net/browse/PKG-5552?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ebf595c8dac3e0fdc4152c51878b498396ec7f30e7a914d6071e674d49420fb8
+  url: https://pypi.io/packages/source/s/sympy/sympy-{{ version }}.tar.gz
+  sha256: 401449d84d07be9d0c7a46a64bd54fe097667d5e7181bfe67ec777be9e01cb13
 
 build:
-  number: 0
+  number: 3
```
Comment (cbouss) [inline recipe/meta.yaml:13]:
```suggestion
  number: 0
```

**Review by ViridianMelody — APPROVED** [review]
Approved once the build number is reset to 0. For context, our build numbers are independent from conda-forge and should be set to 0 for any version new to main.

**Comment by jnoller** [comment]
> Approved once the build number is reset to 0. For context, our build numbers are independent from conda-forge and should be set to 0 for any version new to main.

thanks @ViridianMelody I wasn't sure!

**Comment by skupr-anaconda** [comment]
@jnoller Could you please try to run tests in `test/commands`? See an example https://github.com/sympy/sympy/blob/sympy-1.13.2/.github/workflows/release.yml#L82-L86

**Comment by jnoller** [comment]
@skupr-anaconda Done in https://github.com/AnacondaRecipes/sympy-feedstock/pull/9/commits/12d125c1cfbd6ba2e02e7b1904e876f18e33ae05, verified locally on MacOS

**Review by cbouss — COMMENTED** [review]
Can you still address my previous comment? (I don't know how to add suggestion to non-pull request code.)

To be updated:
mpmath constraints in run to >=1.1.0,<1.4: https://github.com/sympy/sympy/blob/sympy-1.13.2/setup.py#L325

antlr-python-runtime constraints in run_constrained to >=4.11,<4.12
https://github.com/sympy/sympy/blob/sympy-1.13.2/sympy/parsing/latex/_parse_latex_antlr.py#L65

**Comment by jnoller** [comment]
@cbouss Done in https://github.com/AnacondaRecipes/sympy-feedstock/pull/9/commits/5a2a39d5bad0f0e33d1a9834ef65fc726317b80c

**Review by cbouss — APPROVED** [review]

**Pull request merged**